airflow/migrations/versions/0125_2_6_0_add_index_to_task_instance_table.py

```diff
@@ -0,0 +1,50 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Add index to task_instance table
+
+Revision ID: a58082ab9b0e
+Revises: 98ae134e6fff
+Create Date: 2023-04-19 16:48:06.628863
+
+"""
+from __future__ import annotations
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "a58082ab9b0e"
+down_revision = "98ae134e6fff"
+branch_labels = None
+depends_on = None
+airflow_version = "2.6.0"
```

Comment:
```suggestion
airflow_version = "2.6.1"
```
This should be moved to 2.6.1? cc @potiuk

Reply:
yes

Reply:
I have tried to modify the filename and the content of `airflow/migration/versions/0125_2_6_0_add_index_to_task_instance.py`, but the `pre-commit run` command keeps creating the 2_6_0 migration file. Is there a way to make it stop doing this and enforce `2_6_1` in the filename?

Reply:
I think it's because we are in a process of releasing 2.6.0 and we have not updated it yet in main (RC1 has just been cut and we **just** branched off.

Can you update it there please? https://github.com/apache/airflow/blob/main/airflow/__init__.py#L29 And also maybe check if there are other places where 2.6.0 is used.

@ephraimbuddy @pierrejeambrun  -> should we have a step for that somewhere in our process  :)?

Reply:
Changing https://github.com/apache/airflow/blob/main/airflow/__init__.py#L29 produces a bunch of other changes by `pre-commit run`. Some changes are propose by it, for example:

```
...
Check that the REVISION_HEADS_MAP is up-to-date........................................Failed
- hook id: check-revision-heads-map
- exit code: 3

Current airflow version is not in the REVISION_HEADS_MAP
Current airflow version: 2.6.1
Please add the version to the REVISION_HEADS_MAP at: /home/rais/Desktop/workspace/airflow/airflow/utils/db.py

Update version to the latest version in the documentation..............................Failed
- hook id: update-version
- files were modified by this hook
...
```

(btw `2.6.0` is not listed in https://github.com/apache/airflow/blob/main/airflow/utils/db.py#L55, I am not sure how this works).

Should I modify all those files that are produced? I am not sure if this should be done as part of this PR, or someone else should cut a new release? I am not very familiar with this procedure. I don't mind to try, but would need to spam you with questions probably. Please let me know how should I proceed.

Reply:
Hi @ephraimbuddy @potiuk , I see that `2.6.0` has been released, so now I can change https://github.com/apache/airflow/blob/main/airflow/__init__.py#L29 to `2.6.1`? I see it points to `2.7.0-dev`. Not sure how should I patch this for `2.6.1`, is there some part of the documentation that you are aware of which I can look at?

Reply:
You should change it to 2.7.0. If we decide to add it for 2.6 we will cherry-pick it and modify it accordingly in 2.6 branch. It's at the discretion of the release manager, whether it will be included or not. Mainly based on a risk assesment and whether it will not be classified as new feature (in case we consider it risky or a feature -  2.7.0 will be the time it will go out).

You have to do it similarly to this dicussion: https://github.com/apache/airflow/pull/30827#pullrequestreview-1408586127 and whichever of those two changes will be merged first, the other will have to rebase and resolve conflicts after the first merge.

Reply:
Thanks for this, it seems very helpful, will do!

Reply:
OK, I think that's all, I have revised the commit above too be for `2.7.0`. @potiuk @ephraimbuddy please have a look and let me know if it is OK now